staging/src/k8s.io/apiextensions-apiserver/test/integration/conversion/conversion_test.go

```diff
@@ -686,18 +686,22 @@ func expectConversionFailureMessage(id, message string) func(t *testing.T, ctc *
 		objv1beta2 := newConversionMultiVersionFixture(ns, id, "v1beta2")
 		meta, _, _ := unstructured.NestedFieldCopy(obj.Object, "metadata")
 		unstructured.SetNestedField(objv1beta2.Object, meta, "metadata")
-		lastRV := objv1beta2.GetResourceVersion()
 
-		for _, verb := range []string{"get", "list", "create", "update", "patch", "delete", "deletecollection"} {
+		verbs := []string{"get", "create", "update", "patch", "delete", "deletecollection"}
+		// With ResilientWatchcCacheInitialization feature, List requests are rejected with 429 if watchcache is not initialized.
+		// However, in some of these tests that install faulty converter webhook, watchcache will never initialize by definition (as list will never succeed due to faulty converter webook).
+		// In such case, the returned error will differ from the one returned from the etcd, so we need to force the request to go to etcd.
+		if validateList {
```

Comment:
I would like to hear @deads2k opinion on it.

With the ability to serve all requests from watchcache, the mitigation that we did for consistent reads for an ability to force request to be served from etcd:
https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/2340-Consistent-reads-from-cache#what-if-the-watch-cache-is-stale

no longer really works (because watchcache can support any request).

I would like to hear David's confirmation that he is aware of that consequence.

Reply:
pinged David on Slack